lightning/src/ln/functional_test_utils.rs

```diff
@@ -2226,13 +2231,14 @@ pub fn expect_payment_forwarded<CM: AChannelManager, H: NodeHolder<CM=CM>>(
 	}
 }
 
+#[macro_export]
 macro_rules! expect_payment_forwarded {
 	($node: expr, $prev_node: expr, $next_node: expr, $expected_fee: expr, $upstream_force_closed: expr, $downstream_force_closed: expr) => {
 		let mut events = $node.node.get_and_clear_pending_events();
 		assert_eq!(events.len(), 1);
-		$crate::ln::functional_test_utils::expect_payment_forwarded(
-			events.pop().unwrap(), &$node, &$prev_node, &$next_node, $expected_fee,
-			$upstream_force_closed, $downstream_force_closed);
+		$crate::ln::functional_test_utils::expect_payment_forwarded( events.pop().unwrap(), &$node,
```

Comment:
nit:
```suggestion
		$crate::ln::functional_test_utils::expect_payment_forwarded(events.pop().unwrap(), &$node,
```
Though I'd prefer if the params started on the next line per rustfmt, similar elsewhere.